### DISCUSSION TITLE
Extraction of lift and drag files over time

### DISCUSSION THREAD

**Discussion by kor03fluid** [ProjectPhysX/FluidX3D#192] [Q&A]
Is it possible to extract the lift and drag values over time from fluidx3d as a txt file?
If possible, I would like your advice. :)

**Comment by ProjectPhysX** [comment]
Yes. Check the [Ahmed body](https://github.com/ProjectPhysX/FluidX3D/blob/master/src/setup.cpp#L708) sample setup which does exactly that.
